iron-multi-selectable.html

```diff
@@ -46,7 +46,7 @@
     },
 
     observers: [
-      '_updateSelected(selectedValues)'
+      '_updateSelected(selectedValues.*)'
```

Comment:
Sorry, I meant to say `selectedValues.splices` earlier. `selectedValues.*` calls the observer for changes to the array's contents, length, and changes to descendant objects.